### ISSUE TITLE
Template localisation extended to more languages

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4489]
Localisation is needed for human readable dates, as per 
- https://github.com/openlibhums/janeway/issues/4210

In particular, there is a difference in expected order of human readable dates in the US and UK.  Django can handle this automatically through localisation, but currently does not as templates appear to be set to `en` regardless of browser settings.

I propose that `en-gb` and `en-us` be added at minimum, but there is a discussion to be had about what languages should be added in regards to using localisations.

**Comment by ajrbyers** [comment]
I reckon we might need to introduce a Janeway version of the localize template tag that we can use to force localisation even for languages that aren't marked as "supported".

**Comment by StephDriver** [comment]
It would be good to be able to tap into any localisation that django can understand for things like dates, rather than having to specify which one we support.

**Comment by ajrbyers** [comment]
BRF: agreed to close this as we selected a fixed date format.
